spec/classes/datadog_agent_integrations_http_check_spec.rb

```diff
@@ -67,6 +68,7 @@
           threshold: 456,
           window: 789,
           content_match: 'foomatch',
+          reverse_content_match: false,
```

Comment:
can you set this to `true`? `false` and `undef` are falsey, and thus would never render - which is what we're later checking when we do:
```
it { should contain_file(conf_file).with_content(%r{reverse_content_match: true}) }
```

Reply:
Fixed